system/doc/reference_manual/expressions.xml

```diff
@@ -1348,6 +1348,27 @@ handle_call(change, From, #{ state := start } = S) ->
   <section>
     <marker id="bit_syntax"></marker>
     <title>Bit Syntax Expressions</title>
+    <p>The bit syntax operates on <em>bit strings</em>.</p>
+    <p>
+      A <em>bit string</em> is a sequence of bits stored in
+      a sequence of (8-bit) bytes in memory address order.
+      The bits are stored in big-endian format in the byte sequence
+      so the first bit is the higest bit of the first byte.
+      If the length of the <em>bit string</em> is not a multiple
+      of 8 (bits) then the lowest bits of the last byte are unused.
+      A <em>bit string</em> with a length that is a multiple of 8 (bits)
+      is also known as a <em>binary</em>.
+    </p>
+    <p>
+      A <em>bit string</em> can be divided into <em>segments</em>
+      that from left to right are allocated onto the bit sequence
+      in the order above, that is; from high to low bits in each byte,
+      with bytes in memory address order.
+      From this follows that a <em>segment</em> that crosses a byte border
+      does that from the low bit in a byte to the high bit
+      in the subsequent byte.
+    </p>
```

Comment:
I think there should be one or more examples here.

Reply:
Will do